src/ports/postgres/modules/deep_learning/madlib_keras.sql_in

```diff
@@ -253,32 +253,25 @@ CREATE OR REPLACE FUNCTION MADLIB_SCHEMA.internal_keras_predict(
 $$ LANGUAGE plpythonu VOLATILE
 m4_ifdef(`__HAS_FUNCTION_PROPERTIES__', `MODIFIES SQL DATA', `');
 
-CREATE OR REPLACE FUNCTION MADLIB_SCHEMA.madlib_keras_evaluate1(
+CREATE OR REPLACE FUNCTION MADLIB_SCHEMA.madlib_keras_evaluate(
     model_table             VARCHAR,
     test_table              VARCHAR,
-    id_col                  VARCHAR,
-    model_arch_table        VARCHAR,
-    model_arch_id           INTEGER,
-    dependent_varname       VARCHAR,
-    independent_varname     VARCHAR,
-    compile_params          VARCHAR,
-    output_table            VARCHAR
+    output_table            VARCHAR,
+    gpus_per_host           INTEGER
 ) RETURNS VOID AS $$
-    PythonFunctionBodyOnly(`deep_learning', `madlib_keras')
-    with AOControl(False):
```

Comment:
We changed this function to call `PythonFunction` instead of `PythonFunctionBodyOnly` and `PythonFunction` internally calls `with AOControl(False):`. Therefore, removed it from here.